guides/plugins/apps/local-development/app-development-with-docker.md

```diff
@@ -0,0 +1,175 @@
+Simply add this to your `development/docker-compose.yml`.
+```Yaml
+services:
+[...]
+  example_mysql:
+    build: dev-ops/docker/containers/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_USER: app
+      MYSQL_PASSWORD: app
+    ports:
+      - "5506:3306"
+    volumes:
+      - ./dev-ops/docker/_volumes/mysql-example:/mysql-data
+    networks:
+      appSystem:
+        aliases:
+          - appmysql
+```
+
+As you already know you connect your mysql server to the same network as your app server and give it the alias `appmysql`. Furthermore, you can now connect to your database on port `5506` from outside of the docker container.  
+Last but not least we define the credentials for the mysql server and you are done with this.  
+
+### Combining both in one network
+
+Now you need to add your Shopware server to your `development`-network and give him an alias as follows:  
+```Yaml
+services:
+[...]
+  app_server:
+    image: shopware/development:latest
+    networks:
+      shopware:
+        aliases:
+          - docker.vm
+      development:
+        aliases:
+          - shopware
+    extra_hosts:
+      - "docker.vm:127.0.0.1"
+    volumes:
+      - ~/.composer:/.composer
+    tmpfs:
+      - /tmp:mode=1777
+```
+Now your app server can communicate with the Shopware server and your app server can also communicate with your app database.
+
+## Access your app server via ssh
+
+To easily access your app server via ssh, you need to create this script `development/dev-ops/docker/actions/ssh-app-server.sh`.
+```shell script
+#!/usr/bin/env bash
+
+TTY: docker exec -i --env COLUMNS=`tput cols` --env LINES=`tput lines` -u __USERKEY__ -t __EXAMPLE_APP_SERVER_ID__ bash
+```
+This script can be executed from your `development` folder with `./psh.phar docker:ssh-app-server`. Keep in mind that this is only possible when the app server has been started with `./psh.phar docker:start` from your development folder.  
+
+To make sure this script actually knows the ID of your app server which is running in the docker container, you need to define the `EXAMPLE_APP_SERVER_ID` in the `development/.psh.yaml.override`.  
+Your `development/.psh.yaml.override` should look like this:
+```Yaml
+...
+dynamic:
+  ...
+  EXAMPLE_APP_SERVER_ID: docker-compose ps -q example_app_server
+  ...
+...
+```
+
+## Initialising the app server
+To initialise the app server and the app database you should access your app server via ssh and run `composer install --no-interaction`.  
+Next you need to change your `shopwareAppTemplate/.env` and set the DATABASE_URL to `mysql://app:app@appmysql:3306/main`.  
+This should look familiar to you because you just configured it in the `development/docker-compose.yml`.  
+  
+The next steps should be done while connected over ssh to your app server.  
+Now you can set up the database by simply typing `bin/console doctrine:database:create`. This will create your database with name `main`.  
```

Comment:
```suggestion
Now you can set up the database by simply executing `bin/console doctrine:database:create`. This will create your database with the name `main`.  
```